src/Data/Rational/Properties/Heyting.agda

```diff
@@ -0,0 +1,114 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Proof that the rationals form a HeytingField.
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Rational.Properties.Heyting where
```

Comment:
This has now been subsumed into ```Data.Rational.Properties```, but I'm not sure if doing this via anonymous module "jives" with the rest of the file. I took that approach because I didn't want to clutter the primary scope with the various required ```open```s. If you have a suggestion: please let me know.